### PR TITLE
create new instance

### PR DESCRIPTION
**Pull request by sgosline**

### CODE REVIEW

File: config/prod/sjcg-rnaseq-wf.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "NF-OSI"
+  # The resource owner
+  OwnerEmail: "sara.gosline@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-0b898040803850657"      # AWS linux 2 AMI 2.0.20190618 x86_64 HVM gp2
+  # AMIId: "ami-0a313d6098716f372"      # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
```
Comment (thomasyu888) [inline config/prod/sjcg-rnaseq-wf.yaml]: Uncomment the top one and use this line

File: config/prod/sjcg-rnaseq-wf.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  # InstanceType: "m5.4xlarge"
```
Comment (thomasyu888) [inline config/prod/sjcg-rnaseq-wf.yaml]: Need a larger instance and size?

File: config/prod/sjcg-rnaseq-wf.yaml (hunk 1 above)
Comment (sgosline) [reply config/prod/sjcg-rnaseq-wf.yaml]: i dont understand what this means.

File: config/prod/sjcg-rnaseq-wf.yaml (hunk 2 above)
Comment (sgosline) [reply config/prod/sjcg-rnaseq-wf.yaml]: done.

File: config/prod/sjcg-rnaseq-wf.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/sjcg-rnaseq-wf.yaml]: you chose a ubuntu distro (on line 5) the comment says you need to choose an AMIId that matches ubuntu.  you need to comment out the aws linux and uncomment the ubuntu AMI.

Comment (sgosline) [reply config/prod/sjcg-rnaseq-wf.yaml]: got it.

**Pull request merged**